frontend/src/views/FuelSupplies/_schema.jsx

```diff
@@ -175,17 +177,18 @@ export const fuelSupplyColDefs = (optionsData, errors, warnings) => [
     }),
     cellStyle: (params) => {
       const style = StandardCellWarningAndErrors(params, errors, warnings)
-      const conditionalStyle = /other/i.test(params.data.fuelType)
-        ? { backgroundColor: '#fff', borderColor: 'unset' }
-        : { backgroundColor: '#f2f2f2' }
+      const conditionalStyle =
+        params.data.fuelType === FUEL_TYPE_OTHER
```

Comment:
suggestion: DRY this up by wrapping the StandardCellWarningAndErrors into another CellStyle that takes a function to evaluate whether the field is editable or not. This will ensure we have consistent editable / non-editable styles across the product.

Reply:
I have added your suggestion, please review.

Reply:
realized I need to add this to the other schemas

Reply:
now so dry.